libs/common/nest/services/src/lib/mailer/mailer.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+
+import got from 'got';
+
+import { EnvironmentService } from '@tamu-gisc/common/nest/environment';
+import { IMailroomEmailOutbound } from '@tamu-gisc/mailroom/common';
+
+@Injectable()
+export class MailerService {
+  constructor(private readonly env: EnvironmentService) {}
+
+  public sendMail(mailConfig: IMailroomEmailOutbound) {
+    // TODO: Need the environment service to get the address of the mailroom-nest instance from environment file
+    const mailroomUrl = this.env.value('mailroomUrl');
```

Comment:
Is environment service expected to be loaded on the host application? I noticed there's no environment registered on the mailer module.

Reply:
At the moment yes. I would like to integrate the EnvironmentModule in a coming PR so all you have to do is provide the environment variable within the app.module and not worry about importing EnvironmentModule yourself.